Test.C

```diff
@@ -128,20 +128,22 @@ int main(){
     player.currentCredits = 30;
 
     while(player has options and more rounds exists){
-
-        //output player status (if statements)
-        //current credit
-        //current equipment
-        //pulls from info from index amount table referencing index struct table
-                /* WHY DO WE NEED TO USE MULTIPLE ARRAYS FOR EQUIPMENT INSTEAD OF A LIST WITH A QUANTITY ATTRIBUTE? */
-        //current encrypt
-        //iterate and print through current encryption
+        //receive credits NATHAN
+        if(end of round){ //output player status (if statements)
+            player.currentCredits = player.currentCredits - (equipment.cost + encryption.cost); //current credit
+            player.equipmentInventory[i] = player.equipmentInventory[i]; //current equipment
+            /* using i as the array type for equipment here. whatever the current equipment type is will be put in i */
```

Comment:
The comment following this statement is troublesome. "using i as the array type" does not make sense, perhaps it was meant to say "using i as the array index variable"? "whatever the current equipment type is will be put in i" is incorrect. Actually the line `player.equipmentInventory[i] = player.equipmentInventory[i];` does nothing since it just sets the same thing to itself. Also note that `i` has to be declared before hand.

What you want to use here is a for loop to iterate through all of the equipment types and display their amount:
```suggestion
        int i;
        //current equipment
        for (i = 0; i < NUM_EQUIPMENT; i++){
            // details about the equipment type itself is stored in equipmentStock
            // amount of this equipment type that the player has is stored in equipmentInventory
            printf("\n%s: %d", equipmentStock[i]->name, player.equipmentInventory[i]);
        }
```
This loop is pretty bare bones, and the output should be cleaned up, but this should give an idea of what to do.